packages/e2e-test-utils/src/inserter.js

```diff
@@ -110,6 +117,7 @@ export async function searchForBlock( searchTerm ) {
 	await page.focus( INSERTER_SEARCH_SELECTOR );
 	await pressKeyWithModifier( 'primary', 'a' );
 	await page.keyboard.type( searchTerm );
+	await waitForInserterSearch();
```

Comment:
I started looking why you didn't need this in `searchForPattern` and I'm curious why you needed this  in the first place. I mean `insertBlock` waits for the specific block first to be visible.. Is it because we have some tests that only use `searchForBlock`? If that's the case we should also need this in `searchForPattern`, no?

Reply:
>  Is it because we have some tests that only use `searchForBlock`?

Yes, that was my understanding.

> If that's the case we should also need this in `searchForPattern`, no?

Added in https://github.com/WordPress/gutenberg/pull/46153/commits/1fc088cc98957890ae9ba872e30c413796b0d452. It actually makes sense to be there just in case someone uses `searchForPattern()` directly.